components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dto/OAuthConsumerAppDTO.java

```diff
@@ -225,5 +226,17 @@ public String getTokenType() {
     public void setTokenType(String tokenType) {
         this.tokenType = tokenType;
     }
+
+    public boolean isBypassClientCredentials() {
+        return bypassClientCredentials;
+    }
+
+    public boolean getBypassClientCredentials() {
```

Comment:
Cant we use isBypassClientCredentials() method instead of this method?

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1058